paper-dialog-behavior.html

```diff
@@ -140,12 +140,25 @@
       } else {
         this.setAttribute('aria-modal', 'false');
       }
-      // modal implies noCancelOnOutsideClick and withBackdrop if true, don't overwrite
-      // those properties otherwise.
+      // modal implies noCancelOnOutsideClick, withBackdrop and noCancelOnEscKey.
       if (this.modal) {
+        this.__prevNoCancelOnOutsideClick = this.noCancelOnOutsideClick;
+        this.__prevNoCancelOnEscKey = this.noCancelOnEscKey;
+        this.__prevWithBackdrop = this.withBackdrop;
```

Comment:
Do these private members get initialized anywhere? If not, can we initialize them in `created`?